chef/cookbooks/keystone/recipes/ha.rb

```diff
@@ -65,5 +65,12 @@
     only_if { CrowbarPacemakerHelper.is_cluster_founder?(node) }
   end
 
+  pacemaker_order "o-#{service_name}" do
+    ordering "g-haproxy g-postgresql g-rabbitmq cl-#{service_name}"
```

Comment:
Btw, with the order, we should start `cl-keystone` in a notification after the creation of the order.